spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -32,21 +33,37 @@
  * @since 1.0
  *
  */
-public class UnZipResultSplitter {
-
-	public List<Message<Object>> splitUnzippedMap(Map<String, Object> unzippedEntries) {
+public final class UnZipResultSplitter {
```

Comment:
That's not only a reason. Even if it is a new class we should think twice to make it final or package protected. They really may be useful in the future for end user code. Don't forget we develope open source framework, so any code should be available for usage and extension. Only critical infrastructure code may be protected that way.